pkg/databind/pkg/databind/config.go

```diff
@@ -44,18 +44,18 @@ func (y *YAMLConfig) Enabled() bool {
 }
 
 type varEntry struct {
-	TTL         string               `yaml:"ttl,omitempty"`
-	Test        *Test                `yaml:"test,omitempty"`
-	KMS         *secrets.KMS         `yaml:"aws-kms,omitempty"`
-	Vault       *secrets.Vault       `yaml:"vault,omitempty"`
-	CyberArkCLI *secrets.CyberArkCLI `yaml:"cyberark-cli,omitempty"`
-	CyberArkAPI *secrets.CyberArkAPI `yaml:"cyberark-api,omitempty"`
-	Obfuscated  *secrets.Obfuscated  `yaml:"obfuscated,omitempty"`
+	TTL         string               `yaml:"ttl,omitempty" json:"ttl,omitempty"`
```

Comment:
as mentioned in the previous PR, I think we should rename this from YAMLConfig to plain Config since we now serialize in JSON as well

Reply:
I rename the Entry structure in the handler.go. since didn't want to add the greater rename in this PR